fe1-web/Model/Objects/keyPair.ts

```diff
@@ -0,0 +1,36 @@
+import { encodeBase64} from "tweetnacl-util";
+import { PublicKey } from "./publicKey";
+import { PrivateKey } from "./privateKey";
+import { sign } from "tweetnacl";
+import { getStore } from "../../Store/configureStore";
+
+export class KeyPair {
+
+    public static publicKey: PublicKey;
+    public static privateKey: PrivateKey;
+
+    /**
+     * Initialize the keychain with a set of keys
+     */
+    public static initialise() {
```

Comment:
I'd avoid creating a static class here.
It's convenient to have the possibility to represent any number of KeyPairs.

We could however create a separate "KeyStore" singleton, enabling us to retrieve the "default" key pair.